tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -262,6 +288,60 @@ private int WriteTags(ref byte[] bytes, int offset, in SpanModel model, ITagProc
                 }
             }
 
+            // AAS tags need to be set on any span for the backend to properly handle the billing.
+            // That said, it's more intuitive to find it on the local root for the customer.
+            if (AzureAppServices.Metadata.IsRelevant)
+            {
+                // Done here to work both at runtime and in tests.
+                InitializeAasTags();
+                if (model.IsLocalRoot || model.IsChunkOrphan)
+                {
+                    count++;
```

Comment:
You can replace all the `count++` with a single `count += 11` (if I counted correctly).